app/index.html

```diff
@@ -74,7 +74,38 @@
     </div>
 
     <div class="container">
-    <div ng-view=""></div>
+        <div class="feedbackSAM" ng-show="feedbackSAM">
+            <span class="hideButton glyphicon glyphicon-remove" ng-click="feedbackSAM = false"></span>
+
+            <div id="valenceFeedback" class="feedback">
+                <span class="title">How positive is this song?</span>
+                <div class="elements">
+                    <div class="element" ng-repeat="n in [1,2,3,4,5]">
+                        <label for="valenceFeedback{{n}}">
+                            <img width="40px" height="60px" src="/images/valence/valence{{n}}.png" alt="valence{{n}}" />
+                        </label>
+                        <input id="valenceFeedback{{n}}" name="valenceFeedback" type="radio" value="{{n}}"/>
```

Comment:
why no `ng-model="n"`? This allows for a two way binding to value. 
For example:
```js
  angular.module('radioExample', [])
    .controller('ExampleController', ['$scope', function($scope) {
      $scope.color = {
        name: 'blue'
      };
      $scope.specialValue = {
        "id": "12345",
        "value": "green"
      };
    }]);
```
```html
<form name="myForm" ng-controller="ExampleController">
  <label>
    <input type="radio" ng-model="color.name" value="red">
    Red
  </label><br/>
  <label>
    <input type="radio" ng-model="color.name" ng-value="specialValue">
    Green
  </label><br/>
  <label>
    <input type="radio" ng-model="color.name" value="blue">
    Blue
  </label><br/>
  <tt>color = {{color.name | json}}</tt><br/>
 </form>
```
From: https://docs.angularjs.org/api/ng/input/input%5Bradio%5D